cape/cape_result.py

```diff
@@ -2243,7 +2287,30 @@ def process_buffers(
                     ):
                         buffer_body.append(table_row)
                         count_per_source_per_process += 1
-
+                        network_buffers.append((process, api_call, buffer))
+
+    network_buffers_PE = []
+    for process, api, buffer in network_buffers:
+        if any(PE_indicator.decode('ascii') in buffer for PE_indicator in PE_INDICATORS):
+            hash = sha256(buffer.encode()).hexdigest()
+            network_buffers_PE.append((f"{str(process)}-{api}-{hash}", buffer))
+    buffers_PE = []
+    for process, api, buffer in buffers:
+        if any(PE_indicator.decode('ascii') in buffer for PE_indicator in PE_INDICATORS):
+            hash = sha256(buffer.encode()).hexdigest()
+            buffers_PE.append((f"{str(process)}-{api}-{hash}", buffer))
```

Comment:
If we do the check for PE indicators before we add the buffer, we can save time and memory
```suggestion
```